### ISSUE TITLE
Limit how `_` variable can be used

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#309]
# Rule request

## Thesis

Today I saw this code:

```python
[redis_db.delete(symbol_lock_key(_, endpoint_name)) for _ in symbols if True or redis_db.get(symbol_lock_key(_, endpoint_name)) == upid]
```

The thing about it is that we are using `_` as a regular variable.
We also need to list this special case in our `naming.rst`.

## Reasoning

`_` is a special case that indicates that this variable is not used.
So, when it is used we need to alert the user.

**Comment by nixphix** [comment]
I have seen it used like
`
for _ in range(10):
    pass  # do something ten times
`

**Comment by sobolevn** [comment]
Yes, it is. But it is not actually used. It is just defined.

**Comment by nixphix** [comment]
let me add this rule

**Comment by sobolevn** [comment]
@nixphix with pleasure!
